.codecov.yaml

```diff
@@ -0,0 +1,2 @@
+ignore:
+  - "**/zz_generated*.go" # Ignore generated files.
```

Comment:
Please exclude the below:
`**/pkg/apis/**` : CRD related files including generated ones. 
`**/pkg/proto/**` : grpc protobuf client for dapr
`**/pkg/testing/**` : testing mock

Reply:
Done

Reply:
Done